src/UnderlineNav2/UnderlineNav.stories.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react'
+import {EyeIcon} from '@primer/octicons-react'
+import {Meta} from '@storybook/react'
+import UnderlineNav, {UnderlineNavProps} from './index'
+import {BaseStyles, ThemeProvider} from '..'
+import {within, userEvent} from '@storybook/testing-library'
+import {expect} from '@storybook/jest'
+
+export default {
+  title: 'Layout/UnderlineNav',
+  argTypes: {
+    align: {
+      defaultValue: 'left',
+      control: {
+        type: 'radio',
+        options: ['left', 'right']
+      }
+    }
+  },
+  decorators: [
+    Story => {
+      return (
+        <ThemeProvider>
+          <BaseStyles>
+            <Story />
+          </BaseStyles>
+        </ThemeProvider>
+      )
+    }
+  ]
+} as Meta
+
+export const DefaultNav = (args: UnderlineNavProps) => {
+  return (
+    <UnderlineNav {...args}>
+      <UnderlineNav.Link selected>Item 1</UnderlineNav.Link>
+      <UnderlineNav.Link>Item 2</UnderlineNav.Link>
+      <UnderlineNav.Link>Item 3</UnderlineNav.Link>
+    </UnderlineNav>
+  )
+}
+
+// Testing selection
+DefaultNav.play = async ({canvasElement}: {canvasElement: HTMLElement}) => {
```

Comment:
I didn't even know `.play` was a thing! Do we want to keep it in here, or was this only for debugging?

Reply:
These are new interaction tests! I've some minimal documentation in contribution docs about it. I'm working on an ADR for detailed guidelines too. It was my passion week project. They are very similar to integration tests.

Reply:
Ahh very cool! I'm ok with keeping it if you feel like it's ready. 

It would be nice to get that ADR PR opened sooner than later so the rest of the team can understand how to use the new integration tests.